src/components/menu.component.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+import Icon from 'react-native-vector-icons/Ionicons';
+
+import React, {Component, NavigatorIOS, TabBarIOS, View, AlertIOS,} from 'react-native';
```

Comment:
Just fix that and you can merge it into develop.

Btw, I configured this repo so you can't pushed directly to master or develop. So any merges into develop will need to be done via pull requests.